processor/src/fast/call_and_dyn.rs

```diff
@@ -101,6 +101,7 @@ impl FastProcessor {
             current_forest,
         );
 
+        #[cfg(not(feature = "no_err_ctx"))]
         let call_node = current_forest[node_id].unwrap_call();
         let err_ctx = err_ctx!(current_forest, call_node, host);
```

Comment:
This declares `call_node` variable only if `no_err_ctx` feature is off, right? Would that not cause an issue for line 106? If not, maybe worth adding a small comment as to why.

Reply:
https://github.com/0xMiden/miden-vm/pull/2384